src/class/Player/server/NCSPlayer.lua

```diff
@@ -11,6 +11,45 @@ setmetatable(NCSPlayer, {
     __call = function(_, id)
         local self = setmetatable({}, NCSPlayer)
         self.id = id
+        self.name = _G._NCS.Player:getName(self.id)
+        self.license = _G._NCS.Player:getLicense(self.id)
         return (self)
     end
-})
\ No newline at end of file
+})
+
+--- getId
+---@return number
+---@public
+function NCSPlayer:getId()
+    return self.id
+end
+
+--- getName
+---@return string
+---@public
+function NCSPlayer:getName()
+    return self.name
+end
+
+
+--- getLicense
+---@return string
+---@public
+function NCSPlayer:getLicense()
+    return self.license
+end
+
+--- getDimension
+---@return number
+---@public
+function NCSPlayer:getBucket()
+    return GetPlayerRoutingBucket(self.id)
+end
+
+--- setBucket
+---@param bucketID number
+---@return void
+---@public
+function NCSPlayer:setBucket(bucketID)
+    SetEntityRoutingBucket(self.id, bucketID)
```

Comment:
Do you mean `SetPlayerRoutingBucket` ?

Reply:
I think he made a mistake

Reply:
Hello, thank you for your comment. An update has been made to fix this.